src/connectedmachine/azext_connectedmachine/aaz/latest/connectedmachine/_delete.py

```diff
@@ -0,0 +1,144 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "connectedmachine delete",
+    confirmation="Are you sure you want to perform this operation?",
+)
+class Delete(AAZCommand):
+    """Delete operation to delete an Azure Arc-Enabled Server.
```

Comment:
please keep consistent with the help messages of other commands to use verb as the beginning of the message.
```suggestion
    """Delete an Azure Arc-Enabled Server.
```